src/SiteConfig/ConfigBuilder.php

```diff
@@ -341,6 +345,8 @@ public function parseLines(array $lines)
             } elseif ((substr($command, -1) == ')') && preg_match('!^([a-z0-9_]+)\((.*?)\)$!i', $command, $match) && $match[1] == 'replace_string') {
                 array_push($config->find_string, $match[2]);
                 array_push($config->replace_string, $val);
+            } elseif ((substr($command, -1) == ')') && preg_match('!^([a-z0-9_]+)\(([a-z0-9_-]+)\)$!i', $command, $match) && $match[1] == 'http_header' && in_array($match[2], array('user-agent'))) {
```

Comment:
I used `in_array()` to make addition of new headers easier. But I can replace it with a strict check if you prefer.